staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -135,6 +135,17 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	WatchCacheStoreSize = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "store_size",
+			Help:           "Total estimated size of watch cache's store broken by resource type.",
```

Comment:
IIUC, this is effectively only the "all objects" part.
The size of "transaction log" is effectively ignored in this metric.

In non-pathological cases this seems fine, as transaction log is O(75s) old as of now, so most of objects (effectively being pointers to the same objects) are in the original cache.

I think the metric-name reflects that, but maybe let's be a bit more explicit in the `Help` line?